doc/specs/VideoGIFSpec.md

```diff
@@ -0,0 +1,304 @@
+|4 | The inputs can also be adjusted manually via click-and-dragging the displayed tooltips along the highlighted border. | P0 |
+|5 | Dragging the tooltips located on the border’s corners will preserve the region’s current aspect-ratio while adjusting its size. | P0 |
+|6 | Selecting the ‘Confirm’ button from the drop-down menu causes the highlighted border to become locked in as the selection menu transitions to populate with recording options. | P0 |
+
+#### 3.1.7. Recording
+
+|No. | Requirement | Pri
+| - | - | - |
+|1 | Populate the selection menu with recording options: Record, Cancel ([Figure 3.2.3](#figure-323---mock-up-of-video-gif-capture-recording-interface-prior-to-recording)). | P0 |
+|2 | Keep the highlighted border around the selected recording region but remove the dimmed background effect that was created when the selection tool was activated. | P0 |
+|3 | Allow the user to interact with the system as needed to prepare for the initiation of the recording. | P0 |
+|4 | Allow the user to manually modify the recording region prior to the initiation of recording via click-and-drag functionality on the highlighted border like Camtasia’s bounding box in [Figure 1.5.8](#figure-158---camtasias-recording-interface). | P0 |
+|5 | When ‘Play’ is selected, lock the highlighted border so that it can no longer be adjusted. | P0 |
+|6 | When ‘Play’ is selected, overlay the screen with a visual count-down, immediately followed by the initiation of the recording. | P0 |
+|7 |Alternatively start recording by pressing the OS level short-cut: Win + Shift + R. | P0 |
+|8 | When recording begins, transition the recording menu to the active state as depicted in [Figure 3.2.4](#figure-324---mock-up-of-video-gif-capture-recording-interface-during-recording). | P0 |
+|9 | While the recording is active, the user is free to interact with the system as desired. | P0 |
+|10| When the user selects the ‘Stop’ button, the recording is ended and loaded into an editor window like that depicted in [Figure 3.2.5](#figure-325---mock-up-of-video-gif-capture-video-editing). | P0 |
+|11| Alternatively stop the recording and load it into an editor window by pressing the OS level short-cut: Win + Shift + R at any time while recording. | P0 |
+
+#### 3.1.8. Editing
+
+|No. | Requirement | Pri
+| - | - | - |
+|1 | Load the recording from [section 3.1.7](#317-recording) in an editor window as described in [Figure 3.2.5](#figure-325---mock-up-of-video-gif-capture-video-editing). | P0 |
+|2 | A scrub bar should be loaded to trim the recording. Scrubber UI should resemble that of the Photos app in [Figure 1.5.4](#figure-154--trimming-video-via-xbox-game-bar-note-that-the-editing-process-takes-the-user-through-file-explorer-movies--tv-and-photos-before-accessing-the-trimming-functionality). | P0 |
+|3 | On the right-hand menu, display a selection menu for either Video or GIF. | P0 |
```

Comment:
I feel like this should be a dropdown.  Give more flexibility in the future for different formats such as APNG